scripts/mycroft-use.sh

```diff
@@ -110,27 +110,27 @@ function github_init_scripts {
 
         # switch to point a github install and run as the current user
 # TODO Verify all of these
-        sudo sed -i 's_.*SCRIPT=.*_SCRIPT="'${path}'/start.sh audio"_g' /etc/init.d/mycroft-audio
+        sudo sed -i 's_.*SCRIPT=.*_SCRIPT="'${path}'/mycroft-start.sh audio"_g' /etc/init.d/mycroft-audio
```

Comment:
It is actually start-mycroft.sh instead of mycroft-start.sh (see https://github.com/MycroftAI/mycroft-core/blob/dev/start-mycroft.sh)

In this service the script invocation would be ```start-mycroft.sh audio```